README.md

```diff
@@ -2,6 +2,20 @@
 
 # What first?
 
+## Types of VISAs
+### Temporary (Nonimmigrant) Worker
+#### H1B1
```

Comment:
Can you add which countries have access to this visa? I know one is Chile but I always forget the other one